Library/Homebrew/requirements/ruby_requirement.rb

```diff
@@ -44,6 +44,7 @@ def rubies
     if ruby_formula && ruby_formula.installed?
       rubies.unshift ruby_formula.bin/"ruby"
     end
+    rubies.unshift RUBY_PATH unless rubies.include? RUBY_PATH
```

Comment:
I prefer `rubies.push` over `rubies.unshift` to make `RUBY_PATH` a fallback plan rather than the first choice.